react/game/src/app/screens/Profile/components/ProfileEdit/components/ProfileEditForm/index.js

```diff
@@ -1,119 +1,101 @@
-        <Field
-          type="text"
-          validate={[required]}
-          component={Input}
-          name={fields.lastName.name}
-          id={fields.lastName.id}
-          label={fields.lastName.label}
-          placeholder={placeholders.LASTNAME}
-        />
-        <Field
-          type="number"
-          validate={[required, minValueAge, maxValueAge]}
-          component={Input}
-          name={fields.age.name}
-          id={fields.age.id}
-          label={fields.age.label}
-          placeholder={placeholders.AGE}
-        />
-        <Field
-          validate={[required]}
-          component={Textarea}
-          id={fields.aboutMe.id}
-          name={fields.aboutMe.name}
-          label={fields.aboutMe.label}
-          placeholder={placeholders.ABOUT}
-          rows={5}
-          resize={false}
-        />
-        <Field
-          type="text"
-          validate={[required, url]}
-          component={Input}
-          name={fields.profilePicture.name}
-          id={fields.profilePicture.id}
-          label={fields.profilePicture.label}
-          placeholder={placeholders.URL}
-        />
+import ProfileEditFormLayout from './layout';
 
-        <Field
-          type="text"
-          validate={[required, url]}
-          component={Input}
-          name={fields.backgroundPicture.name}
-          id={fields.backgroundPicture.id}
-          label={fields.backgroundPicture.label}
-          placeholder={placeholders.URL}
-        />
+const { fields } = formNames.EDIT_PROFILE;
+const placeholders = formNames.PLACEHOLDERS;
+const formFields = [
+  {
+    type: 'text',
+    validate: [required],
+    component: Input,
+    name: fields.firstName.name,
+    id: fields.firstName.id,
+    label: fields.firstName.label,
+    placeholder: placeholders.NAME
+  },
+  {
+    type: 'text',
+    validate: [required],
+    component: Input,
+    name: fields.lastName.name,
+    id: fields.lastName.id,
+    label: fields.lastName.label,
+    placeholder: placeholders.LASTNAME
+  },
+  {
+    type: 'number',
+    validate: [required, minValueAge, maxValueAge],
+    component: Input,
+    name: fields.age.name,
+    id: fields.age.id,
+    label: fields.age.label,
+    placeholder: placeholders.AGE
+  },
+  {
+    type: 'text',
+    validate: [required],
+    component: Textarea,
+    id: fields.aboutMe.id,
+    name: fields.aboutMe.name,
+    label: fields.aboutMe.label,
+    placeholder: placeholders.ABOUT,
+    rows: 5,
+    resize: false
+  },
+  {
+    type: 'text',
+    validate: [required, url],
+    component: Input,
+    name: fields.profilePicture.name,
+    id: fields.profilePicture.id,
+    label: fields.profilePicture.label,
+    placeholder: placeholders.URL
+  },
+  {
+    type: 'text',
+    validate: [required, url],
+    component: Input,
+    name: fields.backgroundPicture.name,
+    id: fields.backgroundPicture.id,
+    label: fields.backgroundPicture.label,
+    placeholder: placeholders.URL
+  }
+];
```

Comment:
I know that i said to put this const here, but i think it will better extract it in a constant file in this component folder, what u think?

Reply:
Excelent, more clean code!